src/mbgl/layout/symbol_layout.cpp

```diff
@@ -418,6 +418,7 @@ std::unique_ptr<SymbolBucket> SymbolLayout::place(CollisionTile& collisionTile)
 
 template <typename Buffer>
 void SymbolLayout::addSymbols(Buffer &buffer, const SymbolQuads &symbols, float scale, const bool keepUpright, const style::SymbolPlacementType placement, const float placementAngle) {
+    static const uint16_t vertexLength = 4;
```

Comment:
Sure - btw this accounts for the longest type declaration in our codebase: `static constexpr const uint16_t` 😄